lib/web/question_router/get_questions.ts

```diff
@@ -0,0 +1,246 @@
+import prisma from "../../prisma"
```

Comment:
(fyi, useful for changes like these to know if you just moved it to a new file or if there are any changes, it's tricky to notice small changes otherwise)